mmdet/datasets/utils.py

```diff
@@ -127,7 +127,7 @@ def _check_head(self, runner):
         else:
             for name, module in model.named_modules():
                 if hasattr(module, 'num_classes') and not isinstance(
-                        module, (RPNHead, VGG, FusedSemanticHead, GARPNHead)):
+                        module, (RPNHead, VGG, FusedSemanticHead, GARPNHead)) and name != 'backbone':
```

Comment:
What's the purpose of the change?

Reply:
Backbones from `pytorchcv` have class field `num_classes` while local backbones do not have this field. At least it leads to fail in test with `NumClassCheckHook`.